quiz.css

```diff
@@ -0,0 +1,98 @@
+/*quiz.css  1st Quiz - HTML/CSS*/
+
+/* GLOBAL FONT */
+@import url(https://fonts.googleapis.com/css?family=Audiowide|Special+Elite|Cinzel+Decorative:400,700);
+
+body {
+  font-family: "Cinzel Decorative";
+  width: 100%;
+  margin: 0 auto;
+
+}
+.productCards {
+  float: left;
+/*Each card should take up 30% of the width of the browser window.*/
+  width: 30%;
+  margin-left: 2.5%;
+  margin-top: 15px;
+  height: 850px;
+
+/*The entire card has a solid 1px border that is lightblue.*/
+  border: 1px solid lightblue;
+}
+
+/*The card title has a dotted 1px border that is lightgray.*/
+.header {
+  border: 1px dotted lightgray;
+}
+
+/*The title and product image are centered.*/
+.productCards h1 {
```

Comment:
`h1`'s along with all other heading tags should not be qualified. simply use 

```css
h1 {

}
```

>  "Heading elements (h1-h6) should be defined as top-level styles and not scoped to particular areas of the page. The headings are considered to be built-in objects in Object-Oriented CSS, and their appearance should be consistent across an entire site. This allows those styles to be reused across your site for better visual consistency and performance and easier maintenance.

Also... don't over-qualify or over-specify. You only have one type of `h1` on your page so there is no need to reference only the `h1`s inside `.productCards`. Adding more specificity than necessary will make your code difficult to manage as your app gets bigger.